src/test/java/cloud/prefab/client/HarnessServer.java

```diff
@@ -68,16 +68,17 @@ public Response serve(IHTTPSession session) {
           .setSsl(false)
       );
       if (featureFlag) {
-        final Optional<Prefab.FeatureFlagVariant> featureFlagVariant = prefabCloudClient
-          .featureFlagClient()
-          .get(key, Optional.of(user_key), attributes);
-        if (featureFlagVariant.isPresent()) {
-          LOG.info("return {}", featureFlagVariant.get());
-          return newFixedLengthResponse(featureFlagVariant.get().getString());
-        } else {
-          LOG.info("No ff found {}", key);
-          return newFixedLengthResponse("No FF Found");
-        }
+        //        final Optional<Prefab.FeatureFlagVariant> featureFlagVariant = prefabCloudClient
```

Comment:
what's going on here?

Reply:
This was our first pass at client test harness work. It can be cleaned up / removed now that the proper integration test harness is merged.